gateway/src/api/latest.rs

```diff
@@ -220,6 +221,87 @@ async fn request_acme_certificate(
     Ok("Certificate created".to_string())
 }
 
+async fn invalid_project_names(
```

Comment:
Maybe a silly question: why not just create an API endpoint for listing projects and accounts (and their details), then consuming that on the admin CLI to do whatever we need? It would have a greater re-usability for the future, and we could tweak it further in between releases.

Reply:
Yip, this seems better. Thanks for the suggestion D!